js/script.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+    "use strict"
+    
+    document.getElementById("btn-hamburguer").onclick = function(){

```

Comment:
Codacy has a fix for the issue: [Replace `····document.getElementById("btn-hamburguer").onclick·=·function()·{␍` with `··document.getElementById("btn-hamburguer").onclick·=·function·()·{`](https://app.codacy.com/gh/Jeffer48/Consultarq/pullRequest?prid=9456242)
```suggestion
  document.getElementById("btn-hamburguer").onclick = function () {
```